src/core/constraint_template.jl

```diff
@@ -34,6 +34,23 @@ function constraint_tp_voltage(pm::GenericPowerModel; nw::Int=pm.cnw)
 end
 
 
+"Set loose voltage bounds; they will only be set if the voltage magnitude was unconstrained before."
+function constraint_tp_voltage_mag_unbound(pm::GenericPowerModel; nw::Int=pm.cnw, vmin=0.5, vmax=Inf)
+    for bus_id in ids(pm, pm.cnw, :bus)
```

Comment:
Agreed. It would be useful to have some utility functions, where the user can transform a data model by setting standard voltage bounds and so on. Like that, it is also made explicit to the user and should improve awareness of what is going on.